django_project/feti/api_views/occupation.py

```diff
@@ -15,7 +15,7 @@ class OccupationListApiView(APIView):
     Api to list all available occupations.
     """
     def get(self, request):
-        occupations = Occupation.objects.all()
+        occupations = Occupation.objects.only('occupation')
```

Comment:
maybe you can add `.order_by('occupation')` for better view results

Reply:
Okay, I have updated.